src/FFTW.jl

```diff
@@ -35,6 +39,12 @@ function __init__()
         libfftw3[] = MKL_jll.libmkl_rt_path
         libfftw3f[] = MKL_jll.libmkl_rt_path
     end
+
+    @static if !isdefined(Base, :get_extension)
+        Requires.@require ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4" begin
+            include("../ext/FFTWChainRulesCoreExt.jl")
+        end
+    end
```

Comment:
Just always load it outside of `__init__` if `!isdefined(Base, :get_extension)`.